tools/CoqMakefile.in

```diff
@@ -715,7 +715,7 @@ else
 TIMING_EXTRA =
 endif
 
-$(VOFILES): %.vo: %.v
+$(VOFILES): %.vo: %.v | $(VDFILE)
```

Comment:
Yes, this is indeed the right fix. Would be great to get that landed; the currently generated makefiles can lead to really strange behavior.